sidebars.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   someSidebar: {
-    Docusaurus: ['doc1', 'doc2', 'doc3'],
-    Features: ['mdx'],
+    // Docusaurus: ['doc1', 'doc2', 'doc3'],
```

Comment:
Ficou ocmentado, se não for usar pode remover

Reply:
Esse aí deixa assim, que depois eu vou montar a estrutura da documentação, se não vou esquecer como isso funciona. LOL

Reply:
Hahah blz